product_docs/docs/pgd/5/quickstart/further_explore_replication.mdx

```diff
@@ -200,9 +203,8 @@ __OUTPUT__
 (1 row)
 ```
 
-#### Compare the results.
+#### Compare the results
 
-Compare the result from the first and second nodes (kaboom and kaftan) with the result from kaolin. The values will be identical on all three nodes.
+Compare the result from the first and second nodes (kaboom and kaftan) with the result from kaolin. The values are identical on all three nodes.
 
 ## Next steps
```

Comment:
This was missing the content that was meant to follow. Delete? Or link to Next steps topic?

Reply:
We can delete it, as we have the "Next" and "Prev" navigation buttons.